public/app/plugins/datasource/cloudwatch/types.ts

```diff
@@ -386,7 +393,7 @@ export interface VariableQuery extends DataQuery {
   region: string;
   metricName: string;
   dimensionKey: string;
-  dimensionFilters: string;
+  valueDimensions?: Dimensions;
```

Comment:
What is the relationship between `valueDimensions` and `dimensionKey`. Is `dimensionKey` a key of `valueDimensions`?

Is `dimensions` an appropriate name instead of `valueDimensions` here?

Reply:
changed the name back to dimensionFilters